src/ducks/billsMatching/utils.js

```diff
@@ -0,0 +1,261 @@
+/**
```

Comment:
Not necessary to review the whole file in this commit, it is basically a copy/paste from cozy-konnector-libs at this stage.